terraform/modules/VPC-network/main.tf

```diff
@@ -0,0 +1,98 @@
+# VPC
+# New VPC with a /16 CIDR
+resource "aws_vpc" "vpc" {
+  cidr_block           = var.vpcCIDR
+  instance_tenancy     = var.instanceTenancy[0]
+  enable_dns_support   = var.dnsSupport 
+  enable_dns_hostnames = var.dnsHostNames
+    tags = {
+        Name = "${var.project-name}-vpc"
```

Comment:
Nice job, we probably could at this variable in all modules so it becomes easy to track the resources created for this project.